service/src/main/java/aero/minova/cas/service/model/Authorities.java

```diff
@@ -0,0 +1,48 @@
+package aero.minova.cas.service.model;
```

Comment:
In der Standard `application.properties` `pring.jpa.hibernate.ddl-auto=none` angeben,
damit die Datenbank nicht automatisch aktualisiert wird.